### PR TITLE
fix(text): Parse text nodes with marks correctly

### PR DESCRIPTION
**Pull request by Khaledgarbaya**

### CODE REVIEW

File: packages/rich-text-from-markdown/src/index.ts [hunk 1]
```diff
@@ -67,14 +68,13 @@ const nodeContainerTypes = new Map([
   ['heading-4', 'block'],
   ['heading-5', 'block'],
   ['heading-6', 'block'],
-  ['listItem', 'block'],
+  ['list-item', 'block'],
```
Comment (mohamedturki) [inline packages/rich-text-from-markdown/src/index.ts]: Can we use the constants exported by `@contentful/rich-text-types` instead of hard coding them?

Comment (loudmouth) [reply packages/rich-text-from-markdown/src/index.ts]: @mohamedturki done

**Comment by loudmouth** [comment]
@sbezludny and @mohamedturki the requested changes have been made

File: packages/rich-text-from-markdown/src/index.ts (hunk 1 above)
Comment (mohamedturki) [reply packages/rich-text-from-markdown/src/index.ts]: attaboy 👍

**Pull request merged**